bin/cron/analyze_hoc_activity

```diff
@@ -51,10 +51,10 @@ def main
   # https://docs.google.com/document/d/1RTTCpkDYZjqZxfVehkZRkk1HckYMvFdFGs6SEZnK1I8
   total_started += 14_861_327
 
-  today = DateTime.now.to_date
-  day = Date.strptime('2014/12/06', '%Y/%m/%d')
+  today = Date.strptime('2020/02/01', '%Y/%m/%d')
```

Comment:
Could we use the Date constructor, which takes integer year, month, and day arguments?  And same for the next line?
```suggestion
  today = Date.new(2020, 2, 1)
```

Reply:
I can do that :)